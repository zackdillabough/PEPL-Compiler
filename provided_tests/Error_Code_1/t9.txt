POLY A(R) = R + 2 R + 1;
POLY F(Q, W, E, R, T, Y) = Q^121 + W - R^2 - R;
POLY B(R) = R + 2 R + 1;
POLY C(W, C, D, S, J, R, S, F, A, T) = T + 1;

POLY 
	A(R) = R + 2 R + 1;
POLY F(Q, W, E, R, T, Y) = Q^121 + W - R^2 - R;

POLY Y(a, b) = a - b + a^2 - b^2 + 1;

POLY F(Q, W,
 E, R, T, Y) = Q^121 + W - R^2 - R;
POLY B(R) = R + 2 R + 1;

POLY Z(q, w, e, r, t, y) = q - w + e - r + t - y;

POLY A(R) = R + 2 R + 1;
POLY F(Q, W, E, R, T, Y) = Q^121 + W - R^2 - R;
POLY B(
R) = R + 2 R + 1;

POLY YZ = x;

START
C(1, 2, 3, 4, 5, 6, 7, 8, 9, 9);
9 8 6 1 3 4
