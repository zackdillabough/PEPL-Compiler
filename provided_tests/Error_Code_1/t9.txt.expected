Error Code 1: 1 2 3 7 8 12 14 18 19 20 